prisma/schema.prisma

```diff
@@ -144,4 +146,16 @@ model Career {
 
   @@map("careers")
 }
- 
\ No newline at end of file
+
+model Education {
+  id        String    @id @default(cuid())
+  degree    String
+  school    String
+  startDate DateTime @map("start_date")
+  endDate   DateTime? @map("end_date") // in case  not graduated
```

Comment:
if in case they did not graduate, degree can be allow null ?

Reply:
greate, please add ? to startDate, endDate, and also in career model

Reply:
okey